acra/configuring-maintaining/general-configuration/acra-rollback.md

```diff
@@ -0,0 +1,167 @@
+---
+title: acra-rollback
+bookCollapseSection: true
+weight: 10
+---
+
+# acra-rollback
+
+`acra-rollback` is a command-line utility that help you to generate a clean SQL dump from an existing protected one. 
+Rollback utility especially applicable in case of any DB rollback - keys re-generating, going from a Zoneless mode to Zones or vice-versa etc. 
+
+## Command line flags
+
+### General flags
+
+* `--mysql_enable={true|false}`
+
+  Handle MySQL connections. Default is `false`.
+
+* `--postgresql_enable={true|false}`
+
+  Handle PostgreSQL connections. Default is `false`.
+
+* `--client_id=<id>`
+
+  Client ID should be name of file with private key.
+
+* `--connection_string=<connection_string>`
+
+  Connection string for DB in format `dbname=<DBNAME> host=<HOST> port=<PORT> user=<USER> password=<PASSWORD>`.
+
+* `--select=<select_query>`
+
+  Query to fetch data for decryption.
+
+* `--insert=<insert_query>`
+
+  Query to insert decrypted data with placeholders (pg: $n, mysql: ?).
+
+* `--zonemode_enable={true|false}`
+
+  Turn on zone mode. Default is `false`.
+
+* `--execute={true|false}`
+
+  Execute inserts. Default is `false`.
+
+* `--escape={true|false}`
+
+  Escape bytea format. Default is `false`.
+
+
+### Storage destination
+
+#### Filesystem
+
+* `--keys_dir=<path>`
+
+  Path to keystore directory. Default is `.acrakeys`.
```

Comment:
we decided in previous PRs to prefer `folder` noun instead of `directory`. Will be better if you use same content from other utils - https://github.com/cossacklabs/product-docs/blob/master/acra/configuring-maintaining/general-configuration/acra-server.md#keystore